detectable/src/test/java/com/synopsys/integration/detectable/detectables/rubygems/gemlock/functional/RubygemsNodePackagerTest.java

```diff
@@ -51,7 +50,15 @@ void packagerTest() throws MissingExternalIdException {
         final DependencyGraph projects = rubygemsNodePackager.parseProjectDependencies(actualText);
         Assertions.assertEquals(8, projects.getRootDependencies().size());
 
-        GraphCompare.assertEqualsResource("/rubygems/expectedPackager_graph.json", projects);
```

Comment:
The Gemfile.lock used for this test is huge. Assert on everything in that file or remove it.

Reply:
I thought the goal was just to assert on every "unique" type of relationship/stipulation in the file, no, @taikuukaits ?

Reply:
Yes. Do that, and delete every non unique entry.